File: Tecnicell.Server/Models/Entity/UserAccount.cs
using System;
using System.Collections.Generic;

namespace Tecnicell.Server.Models.Entity;

public partial class UserAccount
{
    public string? Name { get; set; }

    public string? Password { get; set; }

    public string UserCode { get; set; } = null!;

    public virtual UserInfo UserCodeNavigation { get; set; } = null!;
}

File: tecnicell.client/src/app/Interfaces/business/Models/UserAccount.ts
import { Image } from "./Image";
import { Role } from "./Role";

export interface UserInfo{
    userCode? : string,
    imageCode? : string,
    name? : string,
    role : string,
    branch :string,
    roleNavigation? : Role,
    imageCodeNavigation? : Image
}

export interface UserAccount{
    userCode? : string,
    name :string,
    password :string
}